### PR TITLE
Use default client for post webhook

### PR DESCRIPTION
**Pull request by Pantani**
Use default client for post webhook to collect metrics for sent observers

### CODE REVIEW

**Comment by catenocrypt** [comment]
👍

File: observer/dispatcher.go [hunk 1]
```diff
@@ -29,23 +25,20 @@ func (d *Dispatcher) dispatch(event Event) {
...
 		"txID":    event.Tx.ID,
 	}
-	go d.postWebhook(webhook, txJson, logParams)
+	go d.postWebhook(webhook, action, logParams)
 	logger.Info("Dispatching webhooks...", logger.Params{"webhook": webhook}, logParams)
 }
 
-func (d *Dispatcher) postWebhook(hook string, data []byte, logParams logger.Params) {
-	_, err := d.Client.Post(hook, "application/json", bytes.NewReader(data))
+func (d *Dispatcher) postWebhook(hook string, data interface{}, logParams logger.Params) {
```
Comment (vikmeup) [inline observer/dispatcher.go]:
```suggestion
func (d *Dispatcher) postWebhook(url string, data interface{}, logParams logger.Params) {
```

File: observer/dispatcher.go [hunk 2]
```diff
@@ -29,23 +25,20 @@ func (d *Dispatcher) dispatch(event Event) {
...
+	client := blockatlas.InitClient(hook)
+	client.Headers["Content-Type"] = "application/json"
+	err := client.Post(nil, "", data)
```
Comment (vikmeup) [inline observer/dispatcher.go]: What is `nil` here?

Comment (Pantani) [reply observer/dispatcher.go]: because the function doesn't need the return object

**Comment by EnoRage** [comment]
Is deprecated

**Pull request closed without merging**